stryker4jvm/src/main/scala/stryker4jvm/logging/FansiLogger.scala

```diff
@@ -0,0 +1,33 @@
+package stryker4jvm.logging
+import fansi.Str
+import stryker4jvm.core.logging.{LogLevel, Logger}
+
+class FansiLogger(val coreLogger: Logger) {
```

Comment:
Maybe a clean solution for the loggers could be to have the same Logger class in core with just `String`s in core, but to have this class (maybe renamed to ColorLogger) extend the base Logger and add the `Fansi` methods and color checking logic. Then the language mutators could have a `Logger` interface passed, while all the projects 'below' that would get a `ColorLogger` injected, which would be the same logger instantation, but just with a 'wider' interface

Reply:
Because fansi.str automatically converts to strings it was unclear at times whether someone was actually using a 'FansiLogger' or a 'core Logger', for this reason it was decided to split it but the proposed solution should work as well.